tests/test_agent.py

```
@@ -1588,6 +1588,22 @@ def test_the_agent_writes_the_name_after_a_restore():
     assert restore_at < naming_at < reboot_at
 
 
+def test_every_lib_file_is_loaded_and_packed():
+    """קובץ lib חדש שאיש לא טוען הוא פונקציה חסרה **בשקט** על מכונה
+    אמיתית — בדיוק הצורה של #84, שם עץ מודולים שלא נארז הפיל
+    ‏used_bytes ואת שם המחשב בלי מילה. שני השערים נבדקים כאן: הבנאי
+    אורז את `lib/*.sh` בגלוב (ולכן קובץ חדש נתפס מאליו), אבל הטעינה
```

Ruff (0.16.2)

[error] Contains control characters that can permit obfuscated code

(PLE2502)